index.html

```diff
@@ -48,35 +48,46 @@
 
           <div class="subscribe-form">
             <h2>Coming to Seattle and Berlin in 2016</h2>
-            <p class="stay-tuned">Stay tuned for more info!</p>
 
-<!--
-             <form>
+             <form id="newsletter-signup" name="newsletter-signup">
               <fieldset>
                 <legend><span>Subscribe for updates</span></legend>
                 <ol>
-                  <li>
-                    <label for="subscribe-email">E-mail</label>
-                    <input type="email" id="subscribe-email" name="email" required placeholder="you@example.com">
+                  <li class="error"></li>
+                  <li id="li_email">
+                    <label for="id_email">E-mail
+                      <input type="email" id="newsletter-email" name="email" required placeholder="you@example.com">
+                    </label>
+                  </li>
+                  <li id="li_newsletters" class="radio">
+                    <legend>I want to hear about...</legend>
```

Comment:
`legend` should be a child of `fieldset`. You can nest a fieldset here to group the two options together:

```
<li>
  <fieldset>
    <legend>I want to hear about…</legend>
    ...
  </fieldset>
</li>
```